### DISCUSSION TITLE
Battery Discharge Limit

### DISCUSSION THREAD

**Discussion by okanano** [mletenay/home-assistant-goodwe-inverter#248] [Q&A]
Hi,

I'm wondering, is there any option to change the parameter Battery Discharge Limit - sensor.battery_discharge_limit?

Model: GoodWe GW5048D-ES
Firmware: 2424J

Thanks!

**Comment by lampra1** [comment]
This is probably controlled by the BMS, based among other things on the SOC and temperature, so I wouldn't expect it to be configurable by the user.
